Sources/View/Widget/SelectionWidget.swift

```diff
@@ -39,7 +39,7 @@ final class SelectionWidget: AbstractWidget {
 
     override func setupLayout(field: HyperwalletField) {
         super.setupLayout(field: field)
-        labelField.accessibilityIdentifier = field.name
+        labelField.accessibilityIdentifier = "\(field.name ?? "")_value"
```

Comment:
keep it as it is. No need of adding value. I'll update the ticket

Reply:
fixed